firefox-suggest-history-vs-offline.toml

```
@@ -19,6 +19,18 @@ from_expression = """(
 window_start = -7
 window_end = 0
 
+[segments.searchsuggestion_users]
+select_expression = """
+  COALESCE((SUM(COALESCE((
+    SELECT SUM(x.value) FROM UNNEST(scalar_parent_urlbar_picked_searchsuggestion_sum
+  ) x), 0)) > 0), FALSE)
+"""
+data_source = "clients_daily"
+
+[segments.data_sources.clients_daily]
+window_start = -7
+window_end = 0
```

```suggestion
data_source = "clients_daily_custom_windows"

[segments.data_sources.clients_daily_custom_windows]
from_expression = "mozdata.telemetry.clients_daily"
window_start = -7
window_end = 0
```

Feel free to rename the data source. It just needs to be called something other than `clients_daily` since that one already exists